schema/User.re

```diff
@@ -1,14 +1,36 @@
+module Summary = {
+  type t = {
+    unread: int,
+    total: int,
+    projects: int,
+    organizations: int,
+  };
+};
+
+module Message = {
```

Comment:
nice, when we introduce the db each entity here will be on its own file
`Message.re` `Notification.re`

Reply:
top